### ISSUE TITLE
Installation Error: `Errors were encountered while processing`

### ISSUE THREAD

**Issue by Aviksaikat** [Eugeny/tabby#9587]
**Describe the problem**:
Error while installing tabby on `kali linux 6.6.15-2kali1 2024.1` 

```sh
$ sudo dpkg -i ~/Downloads/tabby-1.0.207-linux-x64.deb
Selecting previously unselected package tabby-terminal.
(Reading database ... 472934 files and directories currently installed.)
Preparing to unpack .../tabby-1.0.207-linux-x64.deb ...
Unpacking tabby-terminal (1.0.207) ...
dpkg: dependency problems prevent configuration of tabby-terminal:
 tabby-terminal depends on gconf2; however:
  Package gconf2 is not installed.
 tabby-terminal depends on gconf-service; however:
  Package gconf-service is not installed.

dpkg: error processing package tabby-terminal (--install):
 dependency problems - leaving unconfigured
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for desktop-file-utils (0.27-1) ...
Errors were encountered while processing:
 tabby-terminal
```

Funny thing tabby is running even after encountering these errors.
![image](https://github.com/Eugeny/tabby/assets/31238298/47ed921e-15fe-4686-b3df-67a4efd94a6b)

**To Reproduce**:
On kali linux do `sudo dpkg -i ~/Downloads/tabby-1.0.207-linux-x64.deb`

**Comment by justc0de-it** [comment]
The problem occurs on all versions of Ubuntu and distributions based on it

**Comment by Aviksaikat** [comment]
> The problem occurs on all versions of Ubuntu and distributions based on it

I have used tabby in the past on kali but cant remember how I made it work. Also afaik `gconf2`  is outdated and I don't think Debian ships it anymore. I could be wrong though.

**Comment by giggio** [comment]
I'm having the same problem on Ubuntu 24.04. gconf2 is not available since 23.10. I suspect this problem will become more common, with the new LTS out.

**Comment by Eugeny** [comment]
Fixed in the latest release
